test/images/pets/peer-finder/README.md

```diff
@@ -0,0 +1,39 @@
+# Peer finder
```

Comment:
I don't see it is used anywhere. I already see other 2 images `redis` and `zookeeper` already removed part of https://github.com/kubernetes/contrib/pull/2683.